src/View/Helper/HtmlHelper.php

```diff
@@ -299,6 +299,28 @@ public function link($title, $url = null, array $options = []): string
         ]);
     }
 
+    /**
+     * Creates an HTML link from route path string.
+     *
+     * ### Options
+     *
+     * - `escape` Set to false to disable escaping of title and attributes.
+     * - `escapeTitle` Set to false to disable escaping of title. Takes precedence
+     *   over value of `escape`)
+     * - `confirm` JavaScript confirmation message.
+     *
+     * @param string $title The content to be wrapped by `<a>` tags.
+     * @param string $path Cake-relative route path.
+     * @param array $options Array of options and HTML attributes.
+     * @return string An `<a />` element.
+     * @see \Cake\Routing\Router::pathUrl()
+     * @link https://book.cakephp.org/3/en/views/helpers/html.html#creating-links
+     */
+    public function linkFromPath(string $title, string $path, array $options = []): string
+    {
+        return $this->link($title, ['_path' => $path], $options);
```

Comment:
How would links that need parameters work?

Reply:
They would not use this method. They would use normal one.
This is mainly the syntactic sugar entry point for the 70% use case where this applies and where you can use the above speed up tooling as well.

Or do we have an idea how to append params or query strings in such a string, or through options?

Reply:
We could also just add the $params arg into them. One arg more wouldn't be a big issue.